uint8 IDLE=0
uint8 DRIVING=1
uint8 SLOWING=2
uint8 MEASURING=3

uint8 status